### DISCUSSION TITLE
Hide percentage of battery segment

### DISCUSSION THREAD

**Discussion by jos3s** [JanDeDobbeleer/oh-my-posh#586] [Ideas]
I decided to add the drum segment to my [bubbles](https://github.com/JanDeDobbeleer/oh-my-posh/pull/585) theme, but I realized that there is no option to hide the percentage text. I know that for new users not displaying a value next to the icon can be very confusing, but it would be nice to at least have the option to remove the text and leave only the icons

**Comment by JanDeDobbeleer** [comment]
@jos3s as of now you can use the template functionality. Adding `"template": "{{.Icon}}"` to the properties should do the trick.

**Reply by jos3s** [reply]
It worked, I just had to update the module and the repository
